src/blogs/2016-12-30/stryker-0-5-5.pug

```diff
@@ -0,0 +1,59 @@
+                Gone are the days of guessing how far Stryker is!
+                You can use the new reporter by using the reporter `'progress'` in your configuration.
+
+                **The new progress reporter looks like this:**
+                ![new progress reporter image](../../images/blogs/progress-reporter.png)
+
+                The progress bar is updated as Stryker is running. That way you always know how far it is. You even get an estimation on the time remaining.
+
+                Do you still want to use the **old reporter**? It's still there! Just use the repoter `'dots'` in your configuration.
+
+                ### Clear text reporter updated
+                The clear text reporter no longer prints every single test that was executed for a mutant. It now only prints the first three tests and mentions how many more tests were executed.
+                This behavior can be overwritten by using adding the following piece of config to your `stryker.conf.js`:
+                ```
+                clearTextReporter: { 
+                    maxTestsToLog: 25 
+                },
+                ```
+                Don't want to know which tests were executed? Feel free to set the `maxTestsToLog` property to 0.
+
+                ### Karma config parsing
+                With Stryker 0.5.5 we have also updated the [stryker-karma-runner](https://github.com/stryker-mutator/stryker-karma-runner) with the option to parse your `karma.conf.js`!
+                When you use **version 0.3.3 of the stryker-karma-runner** you can specify the location of your karma config file. 
+                By doing this the stryker-karma-runner will parse your config file, which reduces the amount of duplicate config in your stryker.conf.js.
+
+                An updated version of a stryker.conf.js is:
+                ```
+                // Stryker.conf.js
+                module.exports = function (config) {
+                    config.set({
+                        testRunner: 'karma',
+                        testFramework: 'jasmine', // <-- add your testFramework here
+                        karmaConfigFile: 'karma.conf.js' // <-- add your karma.conf.js file here
+                        mutate: [
+                            'src/**/*.js' // <-- mark files for mutation here
+                        ]
+                    });
+                }
+                ```
+
+                As you can see, there is no need to specify which files Stryker should use! You only have to specify which files you want to mutate. 
+                A more detailed explanation of this feature can be found in the [stryker-karma-runner README.md](https://github.com/stryker-mutator/stryker-karma-runner/blob/master/README.md#configuring).
+
+                #### We hope you enjoy this new version of Stryker! We'd love to hear your feedback!
```

Comment:
#### We hope you enjoy this new version of Stryker! We'd love to hear your feedback on [Gitter](https://gitter.im/stryker-mutator/stryker) and/or Twitter (use #strykermutator)!